test/unittests/UtilityTest.php

```diff
@@ -368,6 +368,34 @@ static function fakeGetTestNameByCommentID(
         return $testName;
     }
 
+    /**
+     * Get a list of DDE instruments installed in Loris.
+     *
+     * @param \NDB_Config $config The mock config used for testing
+     *
+     * @return array of the form Test_name => Full Description
+     */
+    static function fakeGetAllDDEInstruments(\NDB_Config $config): array
```

Comment:
I thought that FakeUtility was scrap.  Testing a copy of a class do not test the class itself but a copy of the class at some point in time that is not necessarily in sync.

Reply:
Based on my presentation today and on the new issue #4989 I'm going to try to scrap it today or tomorrow! Stay tuned :)